content/references/client-libraries.ja.md

```diff
@@ -0,0 +1,20 @@
+---
+html: client-libraries.html
+parent: references.html
+blurb: これらのライブラリを使用して、お好みのプログラミング言語からXRP Ledgerにアクセスします。
+---
+# クライアントライブラリ
+
+これらのクライアントライブラリは、XRP Ledgerのデータにアクセスし処理するための一般的な作業の一部を簡素化し、それぞれのプログラミング言語のネイティブな規約に従った形でデータを表示します。
+
+その他のプログラミング言語の場合、[HTTP APIs](http-websocket-apis.html) を通してXRP Ledgerにアクセスすることが可能です。
+
+| 言語 | ライブラリ名 | Get Started | APIリファレンス | ソースコード |
+|----------|--------------|-------------|---------------|-------------|
+| **Python**   | `xrpl-py`      | [Get Started Using Python](get-started-using-python.html) | [API Reference](https://xrpl-py.readthedocs.io/) | [Repo](https://github.com/XRPLF/xrpl-py) |
+| **JavaScript** / **TypeScript** | `xrpl.js` | [Get Started](get-started-using-javascript.html) |  [API Reference](https://js.xrpl.org/) | [Repo](https://github.com/XRPLF/xrpl.js) |
+| **C++**      | `rippled` Signing Library | [Get Started](https://github.com/ripple/rippled/tree/develop/Builds/linux#signing-library) |  | (Part of [`rippled`](https://github.com/ripple/rippled/)) |
+| **Java** | `xrpl4j` | [Get Started Using Java](get-started-using-java.html) | [API Reference](https://javadoc.io/doc/org.xrpl/)  | [Repo](https://github.com/XRPLF/xrpl4j) |
```

Comment:
I notice that for JavaScript the "Get Started" guide's title was previously translated "JavaScriptを使ってみよう", so I think it should also use that name here. It would probably be wise to be consistent when translating the page titles for other programming languages' "Get Started with ____" pages too.